### PR TITLE
Add support for Montreal Forced Aligner

### PR DESCRIPTION
**Pull request by daemon**
- Implement spectrogram a bit more similar to Meyda's
- Add MFA export script
- Implement MFA dataset attachment routines
- Use time stretching augmentation by default

### CODE REVIEW

File: ww4ff/align/vad.py [hunk 1]
```diff
@@ -16,14 +16,14 @@ def detect_vad(frames):
...
-                buf = (frame.numpy() * 32767 / 1.414)
+                buf = frame.numpy() * 32767
                 buf = buf.astype(np.int16).tobytes()
                 if frame.size(0) < frame_len or vad.is_speech(buf, audio.sample_rate):
                     break
                 start += self.frame_ms
             return start
         from ww4ff.data.dataset import AudioClipExample
-        audio = audio  # type: AudioClipExample
+        audio  # type: AudioClipExample
```
Comment (ljj7975) [inline ww4ff/align/vad.py:26]: have 
```
    def align(self, audio: AudioClipExample) -> AlignedTranscription:
```
and remove this line?

File: ww4ff/model/inference.py [hunk 2]
```diff
@@ -15,115 +15,74 @@
 
 
 class InferenceEngineSettings(BaseSettings):
```
Comment (ljj7975) [inline ww4ff/model/inference.py:17]: Can we keep my version to match the browser version?

File: ww4ff/settings.py [hunk 3]
```diff
@@ -24,7 +24,7 @@ class TrainingSettings(BaseSettings):
     device: str = 'cuda:0'
     batch_size: int = 16
     lr_decay: float = 0.75
-    max_window_size_seconds: float = 1
+    max_window_size_seconds: float = 0.75
```
Comment (ljj7975) [inline ww4ff/settings.py:27]: what was this change for `max_window_size_seconds`?

File: ww4ff/align/vad.py (hunk 1 above)
Comment (daemon) [reply ww4ff/align/vad.py:26]: I had that originally, but that results in circular import.

File: ww4ff/settings.py (hunk 3 above)
Comment (daemon) [reply ww4ff/settings.py:27]: I was testing out a different default setting. We still use 1 second (see envs/res8.env).

File: ww4ff/model/inference.py (hunk 2 above)
Comment (daemon) [reply ww4ff/model/inference.py:17]: I'll try out both again, but this wasn't working too well. Maybe my settings weren't optimal.

File: ww4ff/align/vad.py (hunk 1 above)
Comment (ljj7975) [reply ww4ff/align/vad.py:26]: hrm okay. let's get back to this

File: ww4ff/model/inference.py (hunk 2 above)
Comment (ljj7975) [reply ww4ff/model/inference.py:17]: So, I did have noticed that the metrics were too good with the new version.
was that what you mean by working too well?

Comment (daemon) [reply ww4ff/model/inference.py:17]: No, it was just observations using the live demo.

**Pull request merged**